### DISCUSSION TITLE
Key choices and TPMs

### DISCUSSION THREAD

**Discussion by nickray** [charmbracelet/charm#113] [Q&A] [answered]
Cool tool!

I was wondering why RSA is the default algorithm instead of Ed255 (which you already implemented)?

Also, if you supported P256, there could be a nice integration with TPM 2.0 (e.g. with go-tpm), to have keys that are actually bound to the machine they're generated on vs. lying around in `~/.local/share/charm`.

**Comment by bashbunni** [comment, marked as answer]
Hey @nickray 
Could you please explain the RSA issue further? It could be that since you created this issue that we have updated the default key generation to Ed25519, so I'm curious if this is still an issue.

**Reply by nickray** [reply]
I guess this is resolved then; apologies haven't been following this project anymore.

**Reply by bashbunni** [reply]
Ah, ok no problem! Thank you for mentioning it in the first place 😄
